mne/realtime/base_client.py

```diff
@@ -0,0 +1,193 @@
+# Authors: Teon Brooks <teon.brooks@gmail.com>
+#          Mainak Jas <mainak@neuro.hut.fi>
+#
+# License: BSD (3-clause)
+
+import threading
+import time
+import numpy as np
+
+from ..utils import logger, fill_doc
+
+
+def _buffer_recv_worker(client):
+    """Worker thread that constantly receives buffers."""
+    try:
+        for raw_buffer in client.iter_raw_buffers():
+            client._push_raw_buffer(raw_buffer)
+    except RuntimeError as err:
+        # something is wrong, the server stopped (or something)
+        client._recv_thread = None
+        print('Buffer receive thread stopped: %s' % err)
+
+
+class _BaseClient(object):
+    """Base Realtime Client.
+
+    Parameters
+    ----------
+    identifier : str
```

Comment:
Don't we call this `host` elsewhere?

Reply:
it's for the subsequent refactor of the FTClient

Reply:
hmm ... but how would you do it? By deprecating `host` ? It's a bit of an invasive change, no?

Reply:
oh, i see what you mean now, i will call it host. it's common name and it would be expected